cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterPathLookup.java

```diff
@@ -111,13 +111,7 @@ private String getV3RoutingService(String [] uriParts, AllowedMethod requestMeth
       requestMethod.equals(AllowedMethod.GET)) {
       return Constants.Service.APP_FABRIC_HTTP;
     } else if ((uriParts.length >= 4) && uriParts[1].equals("namespaces") && uriParts[3].equals("streams")) {
-      // /v3/namespaces/<namespace>/streams goes to AppFabricHttp
```

Comment:
aren't there any tests in RouterPathTest that need to be updated with this change?